pygmt/src/plot.py

```diff
@@ -122,35 +122,35 @@ def plot(  # noqa: PLR0912
         Draw error bars. Full documentation is at
         :gmt-docs:`plot.html#e`.
     connection : str
-        [**c**\|\ **n**\|\ **r**]\
-        [**a**\|\ **f**\|\ **s**\|\ **r**\|\ *refpoint*].
+        [**c**\|\ **n**\|\ **p**]\
```

Comment:
Please take this opportunity to rewrap the docstrings of **the `connection` parameter** to 88 characters.

Reply:
This is done in commit https://github.com/GenericMappingTools/pygmt/pull/2994/commits/9d4475269003e81751af3462f17aa6d95cadae64.